user/main.py

```diff
@@ -16,7 +18,53 @@ def get_db():
     finally:
         db.close()
 
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+    
+# To insert new user into the table
+
+@app.post("/create", status_code=status.HTTP_201_CREATED)
```

Comment:
Shall we follow the route patterns. Since we are working on the crud. We can use like follows.

List User - "/users" [GET]
Add User - "/users" [POST]
Update User - "/users/{user_id}" [PUT]
Delete User - "/users/{user_id}" [DELETE]

Please update this pattern for all the routes which you have declared. In future works, make sure to apply this.